docs/devices_api.yml

```diff
@@ -1,14 +1,14 @@
 swagger: '2.0'
 info:
   title: Deployments
-  version: '0.1'
+  version: '1'
   description: |
     An API for device firmware deployments. Intended for use by devices.
 
     Devices can get new updates and send information about current deployment status.
 
-host: 'docker.mender.io:8080'
-basePath: '/api/devices/0.1/deployments'
+host: 'docker.mender.io'
```

Comment:
This is just for example, when we have hosted version I would definitely change this real domain we will use to make it copy-paste.